Patches/Ceres/1.10.0/CMakeLists.txt

```diff
@@ -84,12 +84,12 @@ SET(CMAKE_DEBUG_POSTFIX "-debug")
 #
 # For versions without ABI changes, bump the smallest number in CERES_VERSION,
 # but leave the CERES_ABI_VERSION unchanged.
-SET(CERES_VERSION_MAJOR 1)
-SET(CERES_VERSION_MINOR 10)
+SET(CERES_VERSION_MAJOR 2)
```

Comment:
We shouldn't keep this patch for Ceres 1.10 unless it's still needed.  Look at the original patch, so you know what it did:
https://github.com/Kitware/fletch/commit/0052d2d62cdbddd1f69c3e37b1ba7064974f6672

We should apply a similar change to the updated CMakeLists.txt from 2.1 only if needed.  We don't way to take a patched CMakeList.txt from 1.10 and overwrite the file from 2.1

Reply:
Thanks for the catch. It looks like that the flags in that section are nonexistent in the current Ceres version. Should be good to remove